README.md

```diff
@@ -151,7 +151,7 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](https://github.com/Jawad-Nazari/To-Do-List/Add_Remove_Feature/MIT.md) licensed.
+This project is [MIT](https://github.com/Jawad-Nazari/To-Do-List/Intractive_List/MIT.md) licensed.
```

Comment:
- [ ] Everything in your project is working fine, but there is only one issue in your readme: this link is broken. I think it will work by correcting the spellings of `Interactive_List`. Having working links in our readme makes it professional so kindly consider fixing it.

Reply:
Hi @Zuhaib042 ,

Thank you for your time and for reviewing my code. I have fixed the license link. 🙏